<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookMSClass</class>
 <widget class="QMainWindow" name="BookMSClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BookMS</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-220</y>
      <width>671</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/background/library.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>13</y>
      <width>281</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="numberLabel">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;MS Sans Serif&quot;;</string>
           </property>
           <property name="text">
            <string>学工号:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="numberLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="passwordLabel">
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;Microsoft YaHei UI&quot;;
font: 10pt &quot;MS Sans Serif&quot;;</string>
           </property>
           <property name="text">
            <string>密码:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passwordLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="stuButton">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;MS Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>学  生</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="adminButton">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;MS Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>管 理 员</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="loginButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;MS Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>登          录</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BookMS.qrc"/>
 </resources>
 <connections/>
</ui>
